VirtoCommerce.Platform.Data.Azure/AzureBlobProvider.cs

```diff
@@ -36,32 +36,21 @@ public AzureBlobProvider(string connectionString, string cdnUrl)
         /// <summary>
         /// Get blog info by url
         /// </summary>
-        /// <param name="url"></param>
+        /// <param name="blobUrl"></param>
         /// <returns></returns>
-        public virtual BlobInfo GetBlobInfo(string url)
+        public virtual BlobInfo GetBlobInfo(string blobUrl)
         {
-            if (string.IsNullOrEmpty(url))
+            if (string.IsNullOrEmpty(blobUrl))
             {
-                throw new ArgumentNullException(nameof(url));
-            }           
+                throw new ArgumentNullException(nameof(blobUrl));
+            }
 
-            var uri = url.IsAbsoluteUrl() ? new Uri(url) : new Uri(_cloudBlobClient.BaseUri, url.TrimStart('/'));
+            var uri = blobUrl.IsAbsoluteUrl() ? new Uri(blobUrl) : new Uri(_cloudBlobClient.BaseUri, blobUrl.TrimStart('/'));
             BlobInfo retVal = null;
             try
             {
                 var cloudBlob = _cloudBlobClient.GetBlobReferenceFromServer(uri);
-                var fileName = Path.GetFileName(Uri.UnescapeDataString(cloudBlob.Uri.ToString()));
-                var contentType = MimeTypeResolver.ResolveContentType(fileName);
-
-                retVal = new BlobInfo
-                {
-                    Url = GetAbsoluteUrl(cloudBlob.Uri.PathAndQuery),
-                    FileName = fileName,
-                    ContentType = contentType,
-                    Size = cloudBlob.Properties.Length,
-                    ModifiedDate = cloudBlob.Properties.LastModified?.DateTime,
-                    RelativeUrl = cloudBlob.Uri.LocalPath
-                };
+                retVal = ConvertCloudBlobToBlobInfo(cloudBlob);
             }
             catch (Exception)
```

Comment:
why are we hiding this absolute dangerous exception here? I predict that there are might be a lot of problems in caller code. Eg: what will be the real reason that we won't get this blob info? How we'll know about these problems?
The technical details should be at least logged here

Reply:
It`is a refactoring.  It`is not related to the main target of the task.